iati/core/tests/test_schemas.py

```diff
@@ -212,7 +164,7 @@ def test_schema_codelists_add_duplicate(self, schemas_initialised):
         (default_activity_schema, 'element-name-that-does-not-exist', type(None)),
         (default_organisation_schema, 'iati-organisations', etree._Element),
         (default_organisation_schema, 'organisation-identifier', etree._Element),  # The 'organisation-identifier' is defined within the 'iati-organisation' element.
-        (default_organisation_schema, 'sector', type(None))
+        (default_organisation_schema, 'sector', type(None))  # There is no 'sector' element within the organisation schema.
```

Comment:
Valid point, but I'm going to take this one forward in the `get-data-from-schema` integration branch, as this branch relates to flattening the schema in the `__init__` method.